### ISSUE TITLE
Crashes moving schematics with wand when used in conjunction with Minihud

### ISSUE THREAD

**Issue by Syntaxvgm** [sakura-ryoko/litematica#23]
I'm not sure if I should be putting this on the Litematica or Minihud issue tracker, I didn't investigate the issue so I'm not sure where the source is. 

I'm not attaching my logs as this problem doesn't write anything to the logs. It's a hard crash sometimes, sometimes the game catches up after a few minutes but either way there's no logs. I did attach a video though, best I can do. I've also uploaded my minihud and litematica config files. Video shows it crashing when trying to move the schematic. I ditch minihud and restart, where it works fine. Minihud, Litematica, and Malilib are the only mods present. 

GPU is a 7900XT, just in case it's a rendering issue and you need to know that. 

Version of Litematica- 1.21-0.18.999
malilib- 1.21.0.19.999
minihud- 1.21.0.31.999
fabric 0.16.0

[litematica.json](https://github.com/user-attachments/files/16271186/litematica.json)
[minihud.json](https://github.com/user-attachments/files/16271197/minihud.json)
https://drive.google.com/file/d/17ZPyUPbYjFlpfSTOavR5aOB8FWV01btW/view?usp=sharing

**Comment by Syntaxvgm** [comment]
~Actually, to expand on this issue, I notice with just litematica the game doesn't tick at all sometimes to start with, it eventually may catch up.~ I didnt notice for a minute cause I move around like /tick freeze. 


So I'm going to go ahead and attach the logs, this is just litematica logs. 

[logs.txt](https://github.com/user-attachments/files/16271649/logs.txt)


This seems to happen most in SP, but I don't have a problem with multiplayer, though minihud+litematica is still causing crashes and freezes in MP. If you end up needing the world, let me know and I'll send it privately. 

Edit
It turns out this happens with just the vanilla launcher and no mod loader. The game ticks for a while, then doesnt tick for a solid minute. I suspect that something strange is up here with 1.21 and some of my worlds. But either way the litematica+minihud problem i was able to replicate pretty consistently and did have a problem in MP with it too.

**Comment by Syntaxvgm** [comment]
Alright I'm going to close this issue, because even though the crashing only happens with minihud+litemitca, and I was able to replicate in it MP, there's something going on with minecraft where it wont even tick if I load certain region files in converted worlds. I suspect the problems I'm experiencing in MP are also borked region files too, as though the server ticks the crashing only happens in certain regions. 

Though there may be a problem with litematica + minihud of some kind as this bug report is more of a rendering problem and in the process I discovered sometimes the game doesn't even tick in certain region files in 1.21 which is not an issue with the renderer, it's impossible to provide a clean test. Who knows, maybe this vanilla issue is also making the renderer do weird crap.
